src/code/z_play.c

```diff
@@ -164,11 +164,11 @@ s32 Play_SetCameraAtEye(PlayState* this, s16 camId, Vec3f* at, Vec3f* eye) {
     camera->dist = Math3D_Distance(at, eye);
 
     if (camera->focalActor != NULL) {
-        camera->atActorOffset.x = at->x - camera->focalActor->world.pos.x;
-        camera->atActorOffset.y = at->y - camera->focalActor->world.pos.y;
-        camera->atActorOffset.z = at->z - camera->focalActor->world.pos.z;
+        camera->focalActorAtOffset.x = at->x - camera->focalActor->world.pos.x;
```

Comment:
I'm still not a fan of "at" in this name. Maybe `focalActorLookAtOffset`? I'd prefer `focalActorOffset`, but either works.

Reply:
There's two types of offsets at play:
```c
camera->focalActorPosOffset.x = focalActorPosRot.pos.x - camera->focalActorPosRot.pos.x;
camera->focalActorPosOffset.y = focalActorPosRot.pos.y - camera->focalActorPosRot.pos.y;
camera->focalActorPosOffset.z = focalActorPosRot.pos.z - camera->focalActorPosRot.pos.z;
```
and
```c
camera->focalActorAtOffset.x = camera->at.x - focalActorPos->x;
camera->focalActorAtOffset.y = camera->at.y - focalActorPos->y;
camera->focalActorAtOffset.z = camera->at.z - focalActorPos->z;
```
So there has to be some kind of distinction between them. Although exactly how they're used and the difference between them isn't exactly worked out. So I feel `focalActorOffset` might be too general

Other than Init/Copy camera, `focalActorAtOffset` seems to be set through this difference involving `at` except for in `Camera_Fixed2` where it is calculated a different way in using player focusPos instead of at.

I could do `focalActorLookAtOffset`, although I think in the context of `Camera`, `At` should be clear since it's used in nearly every function. And a lot would change if we start transitioning from `At` -> `LookAt`, and I do prefer `At` because of the complex names in camera (it gets messier with `LookAt` everywhere), but I wouldn't rule out some kind of hybrid.